### DISCUSSION TITLE
Update specific lambda

### DISCUSSION THREAD

**Discussion by 1g0rkR** [cargo-lambda/cargo-lambda#604] [General]
Hi there, I would like to update existing lambda via cargo lambda deploy functionality. How could I do it?
The following command renames bootstrap file.
`cargo lambda deploy --iam-role arn:aws:iam::*:* --binary-path ./target/lambda/main/bootstrap --binary-name bootstrap some-function-name
`

**Comment by calavera** [comment]
`cargo lambda deploy FUNCTION_NAME`. The last argument for the command is the function's name that you've deployed. I'm going to add more details about those flags in the docs soon, it looks like people are confused about how to use them. Did you read any article online that talks about `--binary-path` and `--binary-name`?

**Reply by 1g0rkR** [reply]
Yes sir. Actually cargo-lambda cannot find bin file because of specific function name. I've tried to setup --binary-path and --binary-name but the result is just renamed bootstrap to function name.
<img width="749" alt="image" src="https://github.com/cargo-lambda/cargo-lambda/assets/90048102/106937e6-b01f-4e10-969a-9198394e8118">

**Reply by 1g0rkR** [reply]
@calavera Can you please advise parameters to deploy lambda with specific name keeping bootstrap.zip file as it is?

**Reply by calavera** [reply]
there is documentation here: https://www.cargo-lambda.info/commands/deploy.html#working-with-multiple-packages. I don't know what your app does, or how it's constructed, so I cannot give you advice on how to deploy it. If you built the app with Cargo Lambda, and the tool created the `target/lambda/package` directory, `cargo lambda deploy` will work as expected. I don't know what that `reviewserver` is.

**Reply by 1g0rkR** [reply]
The project name and the lambda function name are different. How could I update lambda function via cargo lambda deploy ?

**Reply by 1g0rkR** [reply]
The next command works for me. 
`cargo lambda deploy --binary-name reviewer --binary-path ./target/lambda/main/bootstrap review-server-v2-ReviewServer-jGn4OJArz5W3`
